src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,11 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
```

Comment:
```suggestion
type Props = {
  user: User;
};
```
you have already checked if user is null